docs/connectors/morningstar/security-compare.md

```diff
@@ -0,0 +1,92 @@
+# Security Compare
+
+This type retrieves investment data for multiple securities, making it ideal for comparison.
+
+## How to use SecurityCompare
+
+Use the `SecurityCompareConnector` to load the connector.
+
+Specify the securities in the options along with a Postman environment file for authentication and other parameters such as `currency`.
```

Comment:
A postman file is not a must. In implementations you can also use a session token, or just form fields to pass username and password on. This should be clarified.